### PR TITLE
Added wok area to repository

### PR DESCRIPTION
**Pull request by DyaryRaoof**
Added HTML and styling of 4 working cards.
Added HML and styling of about section.
make the whole design mobile responsive.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,5 +62,199 @@ <h1>
...
+                        <div>
+                            <img src="images/html-icon.png" alt="html icon">
+                            <label>HTML</label>
+                        </div>
+                        <div>
+                            <img src="images/css-icon.png" alt="css icon">
+                            <label>css</label>
+                        </div>
+                    </li>
+                    <li class="flex about-2-3-dropdowns">
```
Comment (dasileker) [inline index.html:239]: - [ ] the language section is not aligned correctly, as you can see this is the original 👇
bb

- And yours is like this 👇, the alignment is a bit off.
nn

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @DyaryRaoof** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @DyaryRaoof, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -64,5 +62,199 @@ <h1>
...
+                        <li class="work-card-top-links-2-3">2015</li>
+                    </ul>
+                    <p>
+                        A daily selection of privately personalized reads;
+                        no accounts or sign-ups required.
+                    </p>
+                    <ul class="work-card-languages-ul flex">
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javaScript</li>
```
Comment (mke2111) [inline index.html:88]: - [ ] [OPTIONAL] Add some padding to the stacks used for each project.
Screenshot from 2021-08-14 02-31-19

**Pull request merged**